t.py

```diff
@@ -0,0 +1,86 @@
+#coding=utf-8
+
+from flask import Flask
+import requests
+import json
+import sys
+import time
+import getpass
+
+USERNAME = "" 
+PASSWORD = ""
+
+s = "" # session
```

Comment:
Don't do that, it should be `None`. And, please read the code [here](https://github.com/plamere/spotipy/blob/master/spotipy/client.py#L79), to see how others work on this problem.